potterscript/quotes.py

```diff
@@ -1,63 +1,176 @@
 import random
 
-def gen_random():
+class quote():
+    """
+    Represents a quote.
+
+    Attributes
+    ----------
+    quote : string
+    Represents the line
+
+    speaker : string
+    The character who spoke the quote.
+
+    novel : string
+    The novel in which the quote was spoken.
+
+    """
+    def __init__(self, quote, speaker='You Know Who', novel=None):
+        self.quote = quote
+        self.speaker = speaker
+        self.novel = novel
+        if speaker not in characters:
+            characters.append(str(speaker))
+        if novel not in novels:
+            novels.append(str(novel))
```

Comment:
We should not append to novels(Person should strictly give the right novel name otherwise an error). This reduces the randomness while trying to get a random quote.